dipy/nn/resdnn_histology_example.py

```diff
@@ -0,0 +1,108 @@
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+from dipy.io.image import load_nifti, save_nifti
+from dipy.io.gradients import read_bvals_bvecs
+from dipy.core.gradients import gradient_table
+from dipy.data import get_fnames
+from dipy.reconst.shm import sf_to_sh, just_sh_basis, sh_to_sf
+from dipy.nn.resdnn_histology import Histo_ResDNN
+from dipy.core.sphere import Sphere, HemiSphere
+from dipy.data import default_sphere
+from dipy.viz import window, actor
+
+
+hardi_fname, hardi_bval_fname, hardi_bvec_fname = get_fnames('stanford_hardi')
+
+data, affine = load_nifti(hardi_fname)
+
+bvals, bvecs = read_bvals_bvecs(hardi_bval_fname, hardi_bvec_fname)
+gtab = gradient_table(bvals, bvecs)
+
+# Extract B0's and obtain a mean B0
+b0_indices = np.where(bvals == 0)
+print('B0 indices found are: {}'.format(b0_indices))
+
+mean_b0 = data[:, :, :, b0_indices]
+mean_b0 = np.squeeze(np.nanmean(mean_b0, axis=4))
+print(mean_b0.shape)
+
+# Detect number of b-values and extract a single shell of DW-MRI Data
+unique_shells = np.unique(bvals)
+print('Number of b-values: {}'.format(np.unique(bvals)))
+
+# Going with the assumption that the first shell would be b0's
```

Comment:
If this is an example, phrase this as an instruction so people can generalize to their data